src/courses/advanced/12.md

```diff
@@ -8,16 +8,21 @@ headerDepth: 3
 ## Next Steps
 
 ### Take the Class Survey
-Take our brief [survey](https://forms.office.com/g/W2xtcV2frW) to give feedback to fuel class improvement.
 
-### Save your work on GitHub
-If you want to save your work in your remote repository in GitHub, you need to use Git commands. You can reference a [Git cheat sheet](https://education.github.com/git-cheat-sheet-education.pdf) or checkout [this Git tutorial](https://learngitbranching.js.org/).
+Please take our brief [survey](https://forms.office.com/g/W2xtcV2frW) to provide feedback and help us improve the class.
```

Comment:
we should make the other classes use this language

Reply:
Issue created https://github.com/mitre/saf-training/issues/280